bindings/python/scripts/parse.py

```diff
@@ -115,28 +115,38 @@ def get_compilation_commands(compilation_database_path, filename):
     return list(compilation_commands[0].arguments)[1:-1]
 
 
-def main():
-    args = utils.parse_arguments(script="parse")
-    for source in args.files:
-        source = utils.get_realpath(path=source)
-
-        index = clang.Index.create()
+def parse_source(source, compilation_database_path=None):
```

Comment:
`parse_file(filename, compilation_database_path)` instead? It'll make this function in line with `get_compilation_commands`. (Doesn't matter which one you change, your own use should be consistent)

Reply:
Done.